dotenv

```diff
@@ -53,3 +53,7 @@ RAILS_SERVE_STATIC_FILES=true
 
 ## Use to enable features only available for development (Optional)
 # RAILS_ENV=development
+
+## DISABLE RECORDINGS IN THE MAIN ROOMS PAGE AS WELL AS THE OPTION TO RECORD IN BIGBLUEBUTTON_ENDPOINT.
+## IT IS FALSE BY DEFAULT 
+# DISABLE_RECORDINGS = true
```

Comment:
Let's invert the logic and rename the variable to make it part of the BIGBLUEBUTTON group to:

BIGBLUEBUTTON_RECORDING_ENABLED = true

Which should be tru by default, even if the variable is not included